test/test_mcmc.py

```diff
@@ -67,37 +67,34 @@ def model(data):
         return p_latent
 
     true_probs = np.array([0.9, 0.1])
-    data = dist.bernoulli(true_probs).rvs(size=(1000, 2), random_state=random.PRNGKey(0))
+    data = dist.bernoulli(true_probs).rvs(size=(1000, 2), random_state=random.PRNGKey(1))
     init_params, potential_fn, transform_fn = initialize_model(random.PRNGKey(2), model, (data,), {})
     init_kernel, sample_kernel = hmc(potential_fn, algo=algo)
     hmc_state = init_kernel(init_params,
```

Comment:
This seems to take a really long time for NUTS on my system with the default `step_size`. How long does this test take to terminate on your system?

Reply:
Hmm, it took 11s for HMC and 15s for NUTS on my CPU. I used jax pip version and might have compiled jaxlib from some commits in the master branch. I am not sure if it affects performance. Looking like Travis also runs well.

Reply:
Hmm..for some reason HMC takes ~ 11 s. for me, but NUTS doesn't seem to terminate in any reasonable amount of time. But since its working on travis, I'll merge this.